admin_manual/configuration_files/encryption_configuration.rst

```diff
@@ -2,35 +2,43 @@
 Encryption Configuration
 ========================
 
-ownCloud includes a server-side encryption application. The Encryption app 
-encrypts all files stored on the ownCloud server, and all files on remote 
-storage that is connected to your ownCloud server. Encryption and decryption are 
-performed on the ownCloud server. All files sent to remote storage (for example 
-Dropbox and Google Drive) will be encrypted by the ownCloud server, and upon 
-retrieval, decrypted before serving them to you and anyone you have shared them 
-with.
-
-.. note:: Encrypting files increases their size by roughly 35%, so you must take 
-   this into account when you are provisioning storage and setting storage 
-   quotas. User's quotas are based on the unencrypted file size, and not the 
-   encrypted file size.
+In ownCloud 8.1 the Encryption app has been changed to the ownCloud Default 
+Encryption Module. It has a number of changes and improvements, including:
+
+* When encryption is enabled, all files are no longer encrypted at user's first 
+  logins because this caused timeouts on large installations. Instead, only 
+  files that are created or updated after encryption has been enabled are 
+  encrypted.
+
+* The  "decrypt all" option in the Personal settings has been removed, also for 
```

Comment:
We should also mention here, that the encryption is now a "all-or-nothing" (is that a german phrase only?) thingy.
Because there are several ways to cause problems with encryption otherwise:

1. Share a folder
2. Disable encryption
3. Move a folder into the shared folder
4. Enable encryption

Result: There are no keys for the newly moved files, because we don't want to run all the encryption logic when it is disabled.

@schiesbn @DeepDiver1975 

ref the 10 comments here: https://github.com/owncloud/core/pull/15880/files#r29142909